tidb-cloud/migrate-incremental-data-from-mysql-using-data-migration.md

```diff
@@ -0,0 +1,224 @@
+
+3. Fill in the target connection profile.
+
+   - **Username**: enter the username of the target cluster in TiDB Cloud.
+   - **Password**: enter the password of the TiDB Cloud username.
+
+4. Click **Validate Connection and Next** to validate the information you have entered.
+
+5. Take action according to the message you see:
+
+    - If you use Public IP or VPC Peering, you need to add the Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any).
+    - If you use AWS Private Link, you are prompted to accept the endpoint request. Go to the [AWS VPC console](https://us-west-2.console.aws.amazon.com/vpc/home), and click **Endpoint services** to accept the endpoint request.
+
+## Step 3: Choose migration job type
+
+To migrate only the incrementable data of the source database to TiDB Cloud, choose **Incremental data migration**. In this case, the migration job does not migrate the existing data of the source database to TiDB Cloud, but only migrates ongoing changes of the source database to TiDB Cloud.
+
+In the **Start Position** area, you can specify the following types of start positions for incremental data migration:
+
+- The time when the incremental migration job starts
+- GTID
+- Binlog file name and position
+
+If you do not specify a start position, the migration job will only migrate the incremental data that is generated in the source database after the migration job starts.
+
+Once a migration job starts, you cannot change the start position.
+
+### The time when the incremental migration job starts
+
+If you select this option, the migration job will only migrate the incremental data that is generated in the source database after the migration job starts.
+
+### Specify GTID
+
+Select this option to specify the GTID of the source database, for example, `3E11FA47-71CA-11E1-9E33-C80AA9429562:1-23`. The migration job will replicate the events excluding the specified GTID set to migrate ongoing changes of the source database to TiDB Cloud.
+
+You can run the following command to check the GTID of the source database:
+
+```sql
+SHOW MASTER STATUS;
+```
+
+### Specify binlog file name and position
+
+Select this option to specify the binlog file name (for example, `binlog.000001`) and binlog position (for example, `1307`) of the source database. The migration job will start from the specified binlog file name and position to migrate ongoing changes of the source database to TiDB Cloud.
+
+You can run the following command to check the binlog file name and position of the source database:
+
+```sql
+SHOW MASTER STATUS;
+```
+
+If there is data in the target database, make sure the binlog position is correct. Otherwise, there might be conflicts between the existing data and the incremental data. If conflicts occur, you need to delete the target database and create a new incremental migration job with the correct binlog position.
```

Comment:
"If conflicts occur, you need to delete the target database and create a new incremental migration job with the correct binlog position."
In this senario ,will the dm job failed and if the user resume the  dm job ,the dm job will start safe mode to replace the conflict records that stored in the target table @lichunzhu

Reply:
When I check the code I think the answer is no. I will do some tests to validate this tomorrow.

Reply:
Updated. PTAL.